src/main/java/com/algosenpai/app/controller/README.md

```diff
@@ -0,0 +1,3 @@
+### Controller
```

Comment:
Whats the difference between this and parser?

Reply:
Parser is for validating the user inputs because the user can input it in various forms. Controller is to handle the logic, take in the input and return the output

Reply:
Controller is for my javafx, parse is only to validate user input because the input can come in any form